src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md

```diff
@@ -0,0 +1,423 @@
+with the user's account allowing them to continue their booking on their desktop device. Not being
+able to transfer experiences can be incredibly jarring to users.
+
+Inform the user of the current state of their data. For example, you could show whether the
+app has synced. Educate them where possible but try not to overburden them with messaging.
+
+## Create inclusive design experiences
+
+When designing seek to be inclusive by providing meaningful design devices, simple language,
+standard iconography, and meaningful imagery that will guide the user to complete the action or task
+rather than hinder their progress.
+
+### Use simple concise language
+
+Good UX is not just about a well designed interface. It includes the path a user takes as well as
+the words used in the app. Avoid tech jargon when explaining the state of the app or
+individual UI components. Consider that the phrase "app offline" might not convey to the user the
+current state of the app.
+
+<figure style="display: inline-block; max-width: 45%;" class="w-figure">
+  <img src="download.png" alt="A download icon is a good example">
+  <figcaption class="w-figcaption">
+    <b>DO</b>: Use language and imagery that describes the action.
+  </figcaption>
+</figure>
+
+<figure style="display: inline-block; max-width: 45%;" class="w-figure">
+  <img src="service-worker-ready.png" alt="A service worker icon is a bad example">
+  <figcaption class="w-figcaption">
+    <b>DON'T</b>: Avoid terms that aren't intelligible to non-technical users.
+  </figcaption>
+</figure>
+
+### Use multiple design devices to create accessible user experiences
+
+Use language, color, and visual components to demonstrate a change of state or current status.
+Solely using color to demonstrate state may not be noticed by the user and may be inaccessible to
+users who suffer from visual disabilities. Also, the instinct for designers is to use grayed UI
+to represent offline, but this can have a loaded meaning on the web. Grayed UI such as input elements 
+on a form also means that an element is disabled. This can cause confusion if you *only*
+use color to depict state.
+
+To prevent misunderstandings, express different states to the user in multiple ways, for example
+with color, labels, and UI components.
+
+<figure style="display: inline-block; max-width: 45%;" class="w-figure">
+  <img src="accessibility_color7_do.png" alt="A good example that uses color and text to show an error.">
+  <figcaption class="w-figcaption">
+    <b>DO</b>: Use a mixture of design elements to convey meaning
+  </figcaption>
+</figure>
+
+<figure style="display: inline-block; max-width: 45%;" class="w-figure">
+  <img src="accessibility_color8_dont.png" alt="A bad example only using color.">
+  <figcaption class="w-figcaption">
+    <b>DON'T</b>: Use color as the sole means to describe what is happening.
+  </figcaption>
+</figure>
```

Comment:
Let's use a [compare component](https://web.dev/handbook/web-dev-components/#compare) here?

Reply:
Applied, here and above.